Makefile

```diff
@@ -0,0 +1,30 @@
+help:
+	@echo "Please use \`make <target>' where <target> is one of:"
+	@echo "  help            to show this message"
+	@echo "  all             to to execute lint and test-coverage"
+	@echo "  install         to install integrade to run integration tests"
+	@echo "  install-dev     to install integrade in editable mode to develop"
+	@echo "                  test cases"
+	@echo "  lint            to lint the source code"
+	@echo "  test            to run integrade's framework unit tests"
+	@echo "  test-coverage   to run integrade's unit tests and measure test"
+	@echo "                  coverage"
+
+all: lint test-coverage
+
+install:
+	pip install .
+
+install-dev:
+	pip install -e .[dev]
+
+lint:
+	flake8 .
+
+test:
+	py.test tests
+
+test-coverage:
+	py.test --verbose --cov-report term --cov=integrade tests
```

Comment:
Seems like there is an issue parsing the coverage report, are you sure its configured correctly?

https://codecov.io/github/cloudigrade/integrade/commit/ff0920a01e68e2db2cfb4358fa9681b0356661b7

Reply:
Maybe because there is no line of code yet? I will double check

Reply:
Perhaps, but I'd expect your tests to be included in coverage too.

Reply:
It seems that the report being empty is the issue. The page you've linked says:

```
There was an error processing coverage reports.

Most common issues

File paths are not resolving.  Reports are empty.  Report has expired.
```

Reply:
Running `make test-coverage` I get a non empty file, but it seems like it only looks at the `__init__` files. Your test file isn't being picked up, maybe you need a `coveragerc` file to define your sources?

Reply:
That test file is for the framework and we don't want to measure coverage for it. We will be looking for code coverage only for automation modules, ones inside integrade.* but excluding everything integrade.tests.*

Reply:
Generally those files are included too (see https://codecov.io/gh/cloudigrade/cloudigrade/tree/master/cloudigrade/account/tests) not as code that is tested, but code that is run. Goal is to make sure you don't have any code in your tests that doesn't actually get run.

Reply:
Makes sense, lemme try adding the /tests package then